src/tests/ftest/util/dmg_utils_params.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+from command_utils_base import \
+    BasicParameter, YamlParameters, TransportCredentials
+
+
+class DmgTransportCredentials(TransportCredentials):
+    """Transport credentials listing certificates for secure communication."""
+
+    def __init__(self):
+        """Initialize a TransportConfig object."""
+        super(DmgTransportCredentials, self).__init__(
+            "/run/dmg/transport_config/*", "transport_config")
+
+
+class DmgYamlParameters(YamlParameters):
+    """Defines the dmg configuration yaml parameters."""
+
+    def __init__(self, filename, name, transport):
```

Comment:
It looks like the description is incorrect. We don't have description for name and transport parameters.

Reply:
Done.